tests/utils/find-articles-from-slugs.test.js

```diff
@@ -2,9 +2,9 @@ import { findArticlesFromSlugs } from '../../src/utils/setup/on-create-page';
 
 it('should correctly find featured articles given a set of requested articles', () => {
     let requestedFeaturedSlugs = [
-        '/quickstart/java-setup-crud-operations',
-        '/how-to/golang-alexa-skills',
-        '/how-to/polymorphic-pattern',
+        'quickstart/java-setup-crud-operations',
```

Comment:
Could we test for a leading slash as well since the other series include them

Reply:
Yep! We still are actually on line 36 here

Reply:
Ah thanks, missed that!